integration-tests/src/test/resources/domain-home-on-pv/create-domain.py

```diff
@@ -116,6 +116,12 @@ def getEnvVar(var):
   cmo.setCluster(cl)
   print('Done setting attributes for Server Template: %s' % templateName);
```

Comment:
This print statement belongs after the NAP creation, since NAP is part of the template too.

Reply:
Yep